chatcommands.py

```diff
@@ -1228,7 +1228,7 @@ def postgone(msg):
     if edited is None:
         raise CmdException("That's not a report.")
 
-    msg.edit(edited)
+    msg._client._do_action_despite_throttling(("edit", msg.id, edited))
```

Comment:
Same here — `messages.edit(msg, edited)`

Reply:
This is the only instance where we edit a message (and this command doesn't work anyways and hasn't been used in forever), so it's probably fine

Reply:
🔥 the command then?

Reply:
I've considered it but I'll leave it in in case someone decides to revamp it. Basically the problem is that the regex used needs to be updated to account for Metasmoke links (yes it's that old)

Reply:
It just doesn’t seem to have any use — the post link is still useful for ♦️s, and AIM will tell you it’s been deleted, even after the two minutes is up.